modules/mono/editor/Godot.NET.Sdk/Godot.NET.Sdk/Godot.NET.Sdk.csproj

```
@@ -14,6 +14,8 @@
     <PackageType>MSBuildSdk</PackageType>
     <PackageTags>MSBuildSdk</PackageTags>
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
+    <Copyright>Copyright (c) 2014-present Godot Engine contributors (see AUTHORS.md).
```

Since I assume `AUTHORS.md` won't be shipped together with the nuget package, I think we can change it to just:
```suggestion
    <Copyright>Copyright (c) 2014-present Godot Engine contributors.
```
That's what we do e.g. for in-engine copyright statements.